src/MISC/fix_eco_force.h

```diff
@@ -0,0 +1,91 @@
+/* ----------------------------------------------------------------------
+ LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+ http://lammps.sandia.gov, Sandia National Laboratories
+ Steve Plimpton, sjplimp@sandia.gov
+
+ Copyright (2003) Sandia Corporation.  Under the terms of Contract
+ DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+ certain rights in this software.  This software is distributed under
+ the GNU General Public License.
+
+ See the README file in the top-level LAMMPS directory.
+ ------------------------------------------------------------------------- */
+
+#ifdef FIX_CLASS
+
+FixStyle(eco/force,FixECOForce)
```

Comment:
if this is so similar to fix orient/XXX, it should be named similarly: how about `fix orient/eco` or `fix orient/ecodf`?
this will make it *much* easier for people to find it and try it out. Also, we are always looking for descriptive names of commands (at least for new contributions) and `eco/force` has many associations but nothing similar to what it does.
consequently with the renaming of the fix style name, also the source files and documentation and examples should be renamed to be consistent with the name.